plugins/available/sudo.plugin.bash

```diff
@@ -0,0 +1,15 @@
+cite about-plugin
+about-plugin 'Toggle sudo at the beginning of the current or the previous command'
```

Comment:
You might want to add a note about hitting the ESC key twice to this description.